packages/backend/src/apps/frappe/index.js

```diff
@@ -0,0 +1,19 @@
+import defineApp from '../../helpers/define-app.js';
+import auth from './auth/index.js';
+import actions from './actions/index.js';
+import setBaseUrl from './common/set-base-url.js';
+import addAuthHeader from './common/add-auth-header.js';
+
+export default defineApp({
+  name: 'Frappe / ERPNext',
```

Comment:
I understand that Frappe is the company, and ERPNext is one of their products. If so, this integration is focused on ERPNext and has nothing to do with Frappe itself directly.

If my understanding is correct, could you please update the app name to `Frappe ERPNext` and its key to `frappe-erpnext`, along with the folder name from `frappe` to `frappe-erpnext`? This would enable us to introduce separate integrations for other Frappe products later.